package.json

```diff
@@ -33,6 +32,7 @@
                 "deno": "./lib-esm/web.js",
                 "edge-light": "./lib-esm/web.js",
                 "netlify": "./lib-esm/web.js",
+                "bun": "./lib-esm/bun.js",
```

Comment:
This is a good idea, I was wary of the confusion that this would bring. 
Should the explicit bun export mean `bun:sqlite` or should it be an entry point that will load `bun:sqlite` / web` / `wss` according to the url ?